### ISSUE TITLE
Blank data

### ISSUE THREAD

**Issue by amitprasad2007** [sonnenglas/amazon-mws-laravel#41]
I am not able to pull data its showing all blank thats it.

**Comment by mehmetuygun** [comment]
What PHP version are you using?

**Comment by amitprasad2007** [comment]
I am using PHP 7.2

**Comment by mehmetuygun** [comment]
what's your server environment? Are you using Windows wamp\xamp?

**Comment by amitprasad2007** [comment]
xampp v3.2.4

**Comment by mehmetuygun** [comment]
the problem must be "ssL certificate problem: unable to get local issuer certificate"

download **cacert.pem** file from here http://curl.haxx.se/docs/caextract.html

then include inside your **php.ini** file
`curl.cainfo = C:\wamp\bin\php\php5.5.12\cacert.pem`

make sure curl extension is enabled.

**Comment by amitprasad2007** [comment]
done all the stuff but the issues still remain the same I am getting only the number of order in this form  { }

**Comment by amitprasad2007** [comment]
moreover i am using xampp serve as localhost not wamp

**Comment by mehmetuygun** [comment]
Yes, this was wamp. You should have changed directory according yourself. Check the Laravel log. Storage/logs you should share us the error if you arr having and the code you are trying to execute.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
